scripts/zones/Riverne-Site_B01/mobs/Book_Browser_Bokabraq.lua

```diff
@@ -0,0 +1,65 @@
+-----------------------------------
+-- Area: Riverne - Site B01
+-- Mob: Book Browser Bokabraq
+-- Type: BLM
+-- Note: NM for quest Go! Go! Gobmuffin!
+-- TODO: Test if the NM runs out of MP, from captures it does not appear to but it was not specifically tested
+-----------------------------------
+local ID = zones[xi.zone.RIVERNE_SITE_B01]
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180) -- 3 minutes
+    mob:setMobMod(xi.mobMod.ALWAYS_AGGRO, 1)
+    mob:setMobMod(xi.mobMod.DETECTION, bit.bor(xi.detects.SIGHT, xi.detects.HEARING)) -- NM aggros to sound+sight
+    mob:setMobMod(xi.mobMod.SOUND_RANGE, 20) -- 20 Yalms
+    mob:setMobMod(xi.mobMod.STANDBACK_RANGE, 16) -- 16 Yalms
+    mob:setMobMod(xi.mobMod.SUPERLINK, ID.mob.SPELL_SPITTER_SPILUSPOK)
+    mob:setMobMod(xi.mobMod.SUPERLINK, ID.mob.SPELL_SPITTER_SPILUSPOK + 1)
+    mob:setMod(xi.mod.REFRESH, 5)
+    mob:setLocalVar('Req', 4) -- 0b100 Used for Quest Progression
```

Comment:
Absolutely not. No quest logic in the mob script.
This is also unneeded anyway, you are handling the onMobDeath on the quest script. Just assign them a bit in the quest script.